lib/ASRouter.jsm

```diff
@@ -772,7 +793,9 @@ class _ASRouter {
       const bundledMessages = await this._getBundledMessages(message, target, trigger, force);
       const action = bundledMessages ? {type: "SET_BUNDLED_MESSAGES", data: bundledMessages} : {type: "CLEAR_ALL"};
       target.sendAsyncMessage(OUTGOING_MESSAGE_NAME, action);
-
+    } else if (message.includeBundle) {
+      const bundledMessages = await this._getBundledMessages(message, target, message.includeBundle.trigger, force);
+      target.sendAsyncMessage(OUTGOING_MESSAGE_NAME, {type: "SET_MESSAGE", data: {...message, bundle: bundledMessages && bundledMessages.bundle}});
```

Comment:
@k88hudson There's a try/catch around a different `target.sendAsyncMessage` a few lines earlier. Is that to avoid async exceptions such as…

https://treeherder.mozilla.org/logviewer.html#/jobs?job_id=243831430&repo=try&lineNumber=12245
```
INFO - TEST-UNEXPECTED-FAIL | browser/components/sessionstore/test/browser_newtab_userTypedValue.js | A promise chain failed to handle a rejection: Message port has been destroyed - stack: sendAsyncMessage@resource://gre/modules/remotepagemanager/MessagePort.jsm:270:13
_sendMessageToTarget@resource://activity-stream/lib/ASRouter.jsm:795:14
async*sendNextMessage@resource://activity-stream/lib/ASRouter.jsm:924:16
```

The test seems to be testing on a bunch of pages including about:welcome and the `await getBundledMessages` might be delaying it enough that the tab has closed destroying the message port. (It's actually the ` else if (message.bundled) {` block's sendAsyncMessage that is throwing, but generally this whole method might want to try/catch ??)